packages/components/src/card/Card.module.css

```
@@ -1,56 +1,108 @@
 @value tokens: "../theme/tokens.css";
-@value --font-family, --border-primary, --layer-02, --layer-hover-02, --layer-selected-02, --focus, --fast-transition, --text-primary, --blue-20 from tokens;
+@value --font-family, --background, --border-primary, --layer-01, --layer-hover-01, --border-secondary, --support-background-success, --support-border-success, --support-background-warning, --support-border-warning, --button-background-icon-hover, --button-background-icon-active, --focus, --z-index-base, --icon-primary from tokens;
 
 .card {
-  position: relative;
   font-family: --font-family;
-  display: flex;
-  flex: 1;
-  flex-direction: column;
-  justify-content: space-between;
-  gap: 1rem;
-  padding: 2rem 1.5rem;
-  border-bottom: 1px solid --border-primary;
-  background-color: --layer-02;
-  transition:
-    transform --fast-transition ease-in-out,
-    background-color --fast-transition ease-in-out;
+  position: relative;
+  background-color: --layer-01;
+  border-bottom: 1px solid --border-secondary;
 
-  &:has([data-hovered]) {
-    background-color: --layer-hover-02;
+  &.horizontal {
+    display: flex;
+    flex-flow: row wrap;
   }
 
-  &:has([data-pressed]) {
-    background-color: --layer-selected-02;
+  &:has(.cardLink:hover) {
+    background-color: --layer-hover-01;
   }
 }
 
-.content {
+.cardContent {
+  padding: 1rem 1rem 1.5rem;
   display: flex;
   flex-direction: column;
-  gap: 0.5rem;
+  gap: 0.5rem 1rem;
+  flex-grow: 1;
+
+  &.horizontal {
+    padding: 0.5rem 1.5rem;
+    flex-direction: row;
+    align-items: center;
+    column-gap: 1.5rem;
+  }
 }
 
-.heading {
-  margin: 0;
-  font-size: 1.5rem;
-  line-height: 2rem;
-  font-weight: 500;
-  color: --text-primary;
+.cardActions {
+  display: flex;
+  flex-direction: row;
+  gap: 0.5rem;
+
+  &.horizontal {
+    margin-left: auto;
+  }
 }
 
-.image {
-  height: auto;
+.cardActionArea {
+  border: 0;
+  padding: 0;
+  text-align: unset;
+  background-color: unset;
+  cursor: pointer;
   width: 100%;
-  object-fit: cover;
+
+  &[data-hovered] {
+    background-color: --button-background-icon-hover;
+  }
+
+  &[data-pressed] {
+    background-color: --button-background-icon-active;
+  }
+
+  &:focus {
+    outline: none;
+  }
+
+  &[data-focused] {
+    box-shadow: --focus;
+
+    @media (forced-colors: active) {
+      outline: 2px solid highlight;
+    }
+  }
+}
+
+.cardImage {
   max-width: 100%;
-  vertical-align: middle;
-  aspect-ratio: 4/3;
+  height: auto;
+
+  &.horizontal {
+    max-width: unset;
+  }
 }
 
-.text {
-  margin-block-start: 0;
-  color: --text-primary;
-  margin: 0;
-  padding: 0;
+.cardLink {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-decoration: none;
+
+  &[data-focused] {
```

```suggestion
  &[data-focused], 
  &:focus {
```

För att stödja andra länkkomponenter